src/main/cpp/feedback/LEDStates.cpp

```diff
@@ -124,71 +124,84 @@ void LEDStates::DiagnosticPattern(frc::DriverStation::Alliance alliancecolor, bo
     {
         m_LEDstring->setSpecificLED(0, m_LEDstring->getColorValues(DragonLeds::RED));
     }
-    if (fintake)
+    if (coralInSensor)
     {
```

Comment:
This might be more readable if you did something like

auto color = coralInSensor ? DragonLeds::YELLOW : DragonLeds::BLACK;
m_LEDstring->setSpecificLED(1,m_LEDstring->getColorValues(color);

Similar for the subsequent ones

There are several existing issues that we should probably capture in stories:
- Bring DriverFeedback, LEDStates, DragonLEDs up to standards (e.g. m_ledString, SetSpecificLED)
- Leverage frc::Color Class colors
- Leverage new addressable led capabilities such as effects/patterns, masks, etc.